hackmcgill/dashboard
# Fix/fixed phone number input

### Tickets:

- HCK-

### List of changes:

- Initialize phone number on input instead of only being able to change phone number after account creation where it's set to default 11111111111

### Type of change:

Please delete options that aren't relevant.


### How did you do this?

deleted some code

### How to test:

make an account with a phone number and check the phone number in the manage account page

### Questions:

### PR Checklist:


### Screenshots:
![image](https://github.com/user-attachments/assets/c020531b-e045-4905-9b08-3336cab37e0d)
![image](https://github.com/user-attachments/assets/d17fbbd7-77e8-4729-8198-fe1c573924eb)
